### PR TITLE
Update environment variables in README

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Removed old variables
Updated list of variables
Documented default values

### How to review 
Does it look correct?

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by MebinAbraham — DISMISSED** [review]
`EQ_MINIMIZE_ASSETS` not mentioned?
Is the idea to document every different env variable the app can use?. If so, I think pretty much everything in settings.py should also be mentioned in the README.

File: README.md [hunk 1]
```diff
@@ -313,39 +313,47 @@ pipenv run pybabel update -i app/translations/messages.pot -d app/translations
...
+| EQ_SERVER_SIDE_STORAGE_DATABASE_DRIVER    | postgresql        |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_HOST      |                   |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_PORT      | 5432              |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_NAME      | digitaleqrds      |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS | 10000             |                                                                                               |
+| EQ_STORAGE_BACKEND                        | datastore         |                                                                                               |
+| EQ_DATASTORE_PROJECT_ID                   | datastore         |                                                                                               |
```
Comment (iwootten) [inline README.md]: This doesn't have a default in settings.py by the looks of things

File: README.md [hunk 2]
```diff
@@ -313,39 +313,47 @@ pipenv run pybabel update -i app/translations/messages.pot -d app/translations
...
+| EQ_DYNAMODB_MAX_RETRIES                   | 5                 |                                                                                               |
+| EQ_DYNAMODB_MAX_POOL_CONNECTIONS          | 30                |                                                                                               |
+| EQ_SUBMITTED_RESPONSES_TABLE_NAME         |                   |                                                                                               |
+| EQ_QUESTIONNAIRE_STATE_TABLE_NAME         |                   |                                                                                               |
+| EQ_SESSION_TABLE_NAME                     |                   |                                                                                               |
+| EQ_USED_JTI_CLAIM_TABLE_NAME              |                   |                                                                                               |
+| EQ_NEW_RELIC_ENABLED                      | False             | Enable New Relic monitoring                                                                   |
```
Comment (iwootten) [inline README.md]: Should EQ_NEW_RELIC_CONFIG_FILE also be here?

File: README.md [hunk 3]
```diff
@@ -313,39 +313,47 @@ pipenv run pybabel update -i app/translations/messages.pot -d app/translations
...
+| NEW_RELIC_LICENSE_KEY                     |                   | Enable new relic monitoring by supplying a New Relic licence key                              |
+| NEW_RELIC_APP_NAME                        |                   | The name to display for the application in New Relic                                          |
```
Comment (iwootten) [inline README.md]: Where are these set? Can't see them in settings.py

**Review by iwootten — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (bitdivision) [reply README.md]: This isn't used anymore AFAICS

File: README.md (hunk 2 above)
Comment (jonnyshaw89) [reply README.md]: Not used so ill remove

File: README.md (hunk 3 above)
Comment (jonnyshaw89) [reply README.md]: They are picked up by the NewRelic agent
https://docs.newrelic.com/docs/agents/python-agent/installation-configuration/python-agent-admin-script

**Review by bitdivision — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
@iwootten @MebinAbraham can you re-review?

**Review by iwootten — APPROVED** [review]

**Review by bitdivision — CHANGES_REQUESTED** [review]
MAX_CONTENT_LENGTH
APPLICATION_VERSION
APPLICATION_VERSION_PATH
SQLALCHEMY_DATABASE_URI

**Comment by jonnyshaw89** [comment]
`EQ_APPLICATION_VERSION` is not an env var, its set within the file by the contents of `EQ_APPLICATION_VERSION_PATH`

**Review by bitdivision — APPROVED** [review]

**Pull request merged**